tests/common/test_transaction.py

```diff
@@ -481,6 +475,15 @@ def test_asset_link_deserialization_with_empty_payload():
     assert asset_link == expected
 
 
+def test_asset_link_from_inputs(utx, transfer_utx):
+    from bigchaindb.common.transaction import AssetLink
+
+    expected = AssetLink(utx.id)
+    asset_link = AssetLink.from_inputs(utx, transfer_utx)
```

Comment:
Not really... but then again, none of the common stuff is actually necessary in the server-side code (ala @sbellem's realization that changing the common's API is actually more driver-related work rather than server-related).